keras_nlp/network_tests/test_load_ckpts.py

```diff
@@ -16,20 +16,18 @@
 import pytest
 import tensorflow as tf
 
-import keras_nlp
-from keras_nlp.models.bert import checkpoints as bert_checkpoints
-from keras_nlp.models.bert import (
-    model_class_by_name as bert_model_class_by_name,
-)
-from keras_nlp.models.bert import vocabularies as bert_vocabularies
+from keras_nlp.models.bert import bert_checkpoints
+from keras_nlp.models.bert import bert_models
+from keras_nlp.models.bert import bert_preprocessing
+from keras_nlp.models.bert import bert_tasks
 
 
 @pytest.mark.slow
 class BertCkptTest(tf.test.TestCase):
     def test_load_bert(self):
-        for checkpoint in bert_checkpoints:
-            bert_variant = bert_checkpoints[checkpoint]["model"]
-            bert_class = bert_model_class_by_name(bert_variant)
+        for checkpoint in bert_checkpoints.checkpoints:
+            bert_variant = bert_checkpoints.checkpoints[checkpoint]["model"]
```

Comment:
A little clunky but perhaps inevitable

Reply:
We could consider just wrapping all the direct structs with some sort of readable function names (unexposed to user).

`bert_checkpoints.list_checkpoints_for_arch()` or `bert_checkpoints.get_checkpoint("bert_based_uncase_en")`.

But definitely not on this PR. And probably after we resolve a little more on the config specification questions.